src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -102,7 +102,7 @@ class SearchControls extends Component {
     const { currentQuery: { facilityType } } = this.props;
 
     switch (facilityType) {
-      case 'benefits':
+      case 'va_benefits_facility':
```

Comment:
Should this be a switch if there is only one specified case and a default? Seems more fitting to just use if/else.